modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/js/OAuth2Client.ts

```diff
@@ -94,7 +105,12 @@ class OAuth2Client {
 					tokenResponse.then((response) =>
 						Liferay.Util.SessionStorage.setItem(
 							sessionKey,
-							JSON.stringify(response),
+							JSON.stringify({
+								...response,
+								expires_after_ms:
```

Comment:
Why are we expiring after 1s?

Reply:
Is not that, the response comes 

600

So I need to convert to milliseconds 

600 * 1000 ( 600.000 ) milliseconds + current time in milliseconds

Reply:
ah, my bad, I didn't see that we get expires_in from the response. I thought you were just arbitrarily using 1000